recipes/qpbo/recipe/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "qpbo" %}
+{% set version = "1.4.3" %}
+{% set sha256 = "3885119cddce875fa36221de4d14b577eb2f1696942274c7dcfcce904288d095" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: qpbo-{{ version }}.tar.gz
+  url: https://github.com/DerThorsten/qpbo/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+
+test:
+  commands:
+    - test -d ${PREFIX}/include/qpbo  # [unix]
+    - test -f ${PREFIX}/include/qpbo/qpbo.hpp  # [unix]
+    - if exist %LIBRARY_PREFIX%\include\qpbo\qpbo.hpp (exit 0) else (exit 1)  # [win]
+
+about:
+  home: https://github.com/DerThorsten/qpbo
+  license: GPL
+  license_family: GPL
+  license_file: LICENCE
+  summary: 'a header only version of Qpbo/ Qpbo solver for discrete graphical models'
+  description: 'a header only version of Qpbo/ Qpbo solver for discrete graphical models'
```

Comment:
If the `description` is redundant, feel free to drop it. Only the `summary` is required. The `description` is a nice to have if it helps explains things further.